resources/frontend_client/index_template.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="{{{language}}}">
```

Comment:
@iethree It's probably mostly a thing in Europe, where the main company is in one country, but have employees from other countries.

Let's go with both approaches as @tsmacdonald (:wave:) mentioned.
```suggestion
<html lang="{{{language}}}" translate="no">
```